pkg/core/system.go

```diff
@@ -282,6 +284,19 @@ func deleteNamespaces(kc *kubectlClient, namespaces []string) error {
 	return nil
 }
 
+func deleteSingleResource(kc *kubectlClient, resourceType string, name string) error {
+	fmt.Printf("Deleting %s/%s resource\n", resourceType, name)
+	deleteLog, err := kc.kubeCtl.Exec([]string{"delete", resourceType, name})
+	if err != nil {
```

Comment:
Since the fact this this resource exists is a bug that will hopefully be fixed, we should avoid logging an error in the case it doesn't exist. Instead we can return the err and push the logging responsibility to the caller.

Reply:
ok, agree on this